sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -290,4 +290,14 @@ class SparkSqlParserSuite extends PlanTest {
           basePlan,
           numPartitions = newConf.numShufflePartitions)))
   }
+
+  test("pipeline concatenation") {
+    val concat = Concat(
+      UnresolvedAttribute("a") ::
+      Concat(UnresolvedAttribute("b") :: UnresolvedAttribute("c") :: Nil) ::
+      Nil)
```

Comment:
Because you do `Concat(exprs.map(expression))`, isn't it `Concat(UnresolvedAttribute("a") :: UnresolvedAttribute("b") :: UnresolvedAttribute("c") :: Nil)`?